pkg/broker/broker.go

```diff
@@ -241,15 +241,15 @@ func (b *Broker) getState(ctx context.Context, orgID string) (*statestorage.Real
 		return nil, errors.Wrap(err, "cannot get API Key by org")
 	}
 
-	return statestorage.Get(ctx, key, b.cfg.AtlasURL, b.cfg.RealmURL, b.logger)
+	return statestorage.Get(ctx, key, b.userAgent, b.cfg.AtlasURL, b.cfg.RealmURL, b.logger)
 }
 
 func (b *Broker) AuthMiddleware() mux.MiddlewareFunc {
 	if b.credentials != nil {
 		return authMiddleware(*b.credentials.Broker)
 	}
 
-	return simpleAuthMiddleware(b.cfg.AtlasURL)
```

Comment:
This is not supported and this branch would never be reached because `main` will error out on missing `credentials`